Brewfile

```diff
@@ -0,0 +1,6 @@
+tap "homebrew/bundle"
+tap "homebrew/cask"
+tap "homebrew/core"
+tap "homebrew/services"
+brew "mongodb-community"
+cask "docker"
```

Comment:
Does this need to be in here? Seems OS X dev specific; I'm not strongly opposed to it, but I would love to talk it through a bit.

Reply:
it's absolutely OS X specific; i had meant it as a convenience for developers on Macs

Reply:
Docker is absolutely not my strong suit - what's the usual way to handle OS-specific convenience packages? Is it good practice to have them? Can there be a check made to see what the host OS is?